server/server.go

```diff
@@ -140,13 +140,7 @@ func (s *Server) loadContainer(id string) error {
 		return err
 	}
 
-	var img *pb.ImageSpec
-	image, ok := m.Annotations[annotations.Image]
-	if ok {
-		img = &pb.ImageSpec{
-			Image: image,
-		}
-	}
+	img, _ := m.Annotations[annotations.Image]
```

Comment:
I believe you're correct, but it seems like better practice to verify whether it's there (at some point, we're probably going to want to error out if it isn't present to allow supporting restarting containers).